### ISSUE TITLE
Issue in demo_ycb.sh

### ISSUE THREAD

**Issue by Guru-Uni-siegen** [ethnhe/PVN3D#36]
Hello,
Thanks a lot for the code. I am getting segmentation fault after running demo_ycb.sh. Below is error on my prompt, Any Idea what I could be missing?
 
sh demo_ycb.sh 
/is/sg2/ghegde/Project/PVN3D/pvn3d/common.py:134: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.lm_r_lst = yaml.load(lm_r_file)
test_dataset_size:  2949
loading pretrained mdl.
==> Loading from checkpoint 'train_log/ycb/checkpoints/ycb_pvn3d_best.pth.tar'
==> Done
val: 0it [00:00, ?it/s]/is/sg2/ghegde/install/conda/envs/pcl/lib/python3.6/site-packages/torch/nn/functional.py:2457: UserWarning: nn.functional.upsample is deprecated. Use nn.functional.interpolate instead.
  warnings.warn("nn.functional.upsample is deprecated. Use nn.functional.interpolate instead.")
/is/sg2/ghegde/install/conda/envs/pcl/lib/python3.6/site-packages/torch/nn/functional.py:2539: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  "See the documentation of nn.Upsample for details.".format(mode))
/is/sg2/ghegde/install/conda/envs/pcl/lib/python3.6/site-packages/torch/nn/modules/container.py:92: UserWarning: Implicit dimension choice for log_softmax has been deprecated. Change the call to include dim=X as an argument.
  input = module(input)
Segmentation fault

**Comment by Guru-Uni-siegen** [comment]
GCC < 4.9 version issue.
